### DISCUSSION TITLE
Display color profile

### DISCUSSION THREAD

**Discussion by benoe77** [jrandiny/yoga-slim7-ubuntu#38] [General]
I'm using this
[AUO_B140HAN06_8.zip](https://github.com/jrandiny/yoga-slim7-ubuntu/files/6561318/AUO_B140HAN06_8.zip)
profile for my glossy AU Optronics B140HAN06.8 display.

I downloaded it from here: [https://www.notebookcheck.net/The-Ryzen-7-4800U-is-an-Absolute-Monster-Lenovo-Yoga-Slim-7-14-Laptop-Review.456068.0.html#toc-display-freesync-on-a-laptop](url)

Does anyone have a profile for other displays maybe?

**Comment by skfieldsock** [comment]
yoga pipa
